src/System.ValueTuple/ref/Configurations.props

```diff
@@ -4,8 +4,6 @@
     <BuildConfigurations>
       netcoreapp;
       uap;
-      netstandard;
-      netstandard1.0;
```

Comment:
Currently it was already disabled in this project for anything but netcoreapp and uap. I will consider re-enabling it and using the